opendebates/settings.py

```diff
@@ -260,3 +261,7 @@ def __contains__(self, item):
 DBBACKUP_STORAGE_OPTIONS = {'location': '.'}
 DBBACKUP_FILENAME_TEMPLATE = 'local/{datetime}.{extension}'
 DBBACKUP_SEND_EMAIL = False
+
+# Dummy settings for test etc
+NORECAPTCHA_SITE_KEY = ''
+NORECAPTCHA_SECRET_KEY = ''
```

Comment:
Wondering what the least intrusive way to do this for local development is... 

Hmm, looks like Google provides a fake sitekey and secret: https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-test-with-recaptcha-v2-how-should-i-do (2nd question) That seems to work.